frontend/src/clients/report/CSVClient.ts

```diff
@@ -4,7 +4,7 @@ import dayjs from 'dayjs'
 import { downloadCSV } from '@src/utils/util'
 
 export class CSVClient extends HttpClient {
-  parseTimeStampToHumanDate = (csvTimeStamp: number | undefined): string => dayjs(csvTimeStamp).format('SSS')
+  parseTimeStampToHumanDate = (csvTimeStamp: number | undefined): string => dayjs(csvTimeStamp).format('HHmmSSS')
```

Comment:
We want to try our best to add more numbers to avoid conflicts of downloading csv file.